Cargo.toml

```diff
@@ -32,6 +32,11 @@ serde_yaml = { version = "0.8", optional = true }
 toml = { version = "0.5", optional = true }
 stringly_conversions = { version = "0.1.1", optional = true, features = ["alloc"] }
 
+# avoid building criterion in 1.41.1 CI
+[target.'cfg(bench)'.dev-dependencies]
+criterion = "0.2.11"
+softposit = "0.3.9"
```

Comment:
workaround when one wants to run criterion while supporting old rust compiler

https://github.com/rust-lang/cargo/issues/1596#issuecomment-962730780